release/publishJars.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh -e
+
+printUsage() {
+   echo "Usage : "
+   echo "./publishJars.sh JAMES_SOURCES_PATH JAMES_DIST_PATH RELEASE SHA1 KEY"
+   echo "    JAMES_SOURCES_PATH: path of the James code"
+   echo "    JAMES_DIST_PATH: path of the James dist svn repo"
+   echo "    RELEASE: release version ex.: 3.3.0"
+   echo "    SHA1: sha1 corresponding of the git commit of the release"
+   echo "    KEY: key footprint to use for signing"
+   exit 1
+}
+
+addAndSign() {
+    sha1sum $1 > $1.sha1
+    gpg -u $KEY --output $1.asc --detach-sig $1
+
+    svn add $1
+    svn propset svn:mime-type application/octet-stream $1
+    svn add $1.sha1
+    svn add $1.asc
+    svn propset svn:mime-type application/octet-stream $1.asc
+}
+
+guiceProduct() {
+    mkdir -p $1/target/$2-$RELEASE
```

Comment:
I bet you did not try with $1 containing spaces ;)

Reply:
C'mon, it's a committer tool; they should be aware of that.

Reply:
aware of what? That you are not allowed to work in a /home/me/My Dev/ directory?